src/expression/parse.js

```diff
@@ -1192,6 +1193,22 @@ export const createParse = /* #__PURE__ */ factory(name, dependencies, ({
     return parsePow(state)
   }
 
+  /**
+   * nullish coalescing operator
+   * @return {Node} node
+   * @private
+   */
+  function parseNullishCoalescing (state) {
```

Comment:
Small point: this function should be after parsePow, since the functions here are arranged in order of increasing precedence.